### ISSUE TITLE
virt-manager - SSH -x not working

### ISSUE THREAD

**Issue by tomkoi** [Eugeny/terminus#2907]
Hello i use terminus to connected to my VDI at work and open tmux

i install virt-manager to use for servers with KVM option

 I ssh to servers with KVM image and want to run the virt-manager and got this error:

 
 SSH   X  Could not connect to the X server :6000: Error: connect ECONNREFUSED 127.0.0.1:6000di-330" 14:03 17-Aug-20
 SSH   X  Could not connect to the X server :6000: Error: connect ECONNREFUSED 127.0.0.1:6000

 SSH   X  Could not connect to the X server :6000: Error: connect ECONNREFUSED 127.0.0.1:6000
(virt-manager:34936): Gtk-WARNING **: 14:03:39.149: cannot open display: localhost:10.0

And the virt-manager console didn’t open

i tested this on other app like vnc viewer or mobaxtream and it's working fine

You think you can fix it?

**Comment by Eugeny** [comment]
Have you enabled X11 forwarding in the connection settings?

**Comment by tomkoi** [comment]
Yes
[cid:image003.png@01D674A0.DDB4EEA0]

Best regard

Tom Koifman
MTR labs IT Administrator, Mellanox Labs Operation
• [Office] +972(74)7129340  • [Mobile] +972(54)8119335   • [E-Mail] tomko@nvidia.com<mailto:tomko@nvidia.com>
[cid:image002.png@01D674A0.DDA7CF40]<https://www.nvidia.com/>

**Comment by tomkoi** [comment]
![Annotation 2020-08-17 150724](https://user-images.githubusercontent.com/69791080/90394468-635fd900-e09b-11ea-8914-7c8b3d6e2154.png)

**Comment by hqabcxyxz** [comment]
> Have you enabled X11 forwarding in the connection settings?

I meet the same question, what should I do?

**Comment by tomkoi** [comment]
Yes , anyway i found other app for now to use

Best regard

Tom Koifman

MTR labs IT Administrator, Mellanox Labs Operation

[Office] +972(74)7129340
[Mobile] +972(54)8119335
[E-Mail] tomko@nvidia.com

NVIDIA

________________________________

**Comment by max1c** [comment]
Same problem:
  SSH   X  Could not connect to the X server :6000: Error: connect ECONNREFUSED
  127.0.0.1:6000
  xterm: Xt error: Can't open display: localhost:52.0

Seems to work sometimes but other times gives this error. Not sure what the issue is but looks like there are open issues about this.

**Comment by Eugeny** [comment]
Do you have an X server running on your client?

**Comment by max1c** [comment]
> Do you have an X server running on your client?

It's enabled in the connection settings if that's what you mean. 
![image](https://user-images.githubusercontent.com/11587156/107150885-00564d00-692e-11eb-9782-4a8d96a80073.png)
I found that it works sometimes and others it gives this error.

**Comment by Eugeny** [comment]
I mean an X server like vcxsrv or Xming that you need to start separately from Terminus (if you're on Window)

**Comment by max1c** [comment]
> I mean an X server like vcxsrv or Xming that you need to start separately from Terminus (if you're on Windows)

Nope, I was assuming that it would be included with the terminus itself. But I guess that would explain it.

**Comment by max1c** [comment]
@Eugeny Can Xming be added to the terminus distribution so users don't have to download and install it separately? This would be incredibly helpful.

**Comment by Eugeny** [comment]
@max1c I'd rather not force the users to use a specific server

**Comment by max1c** [comment]
> @max1c I'd rather not force the users to use a specific server

Could it possibly be a separate release? E.g. terminus-1.0.139-Xming-setup.exe 
Also, it could only fallback on it if there is nothing else available. I have been suggesting Terminus to some of the users these days and the lack of built-in X11 server makes things very confusing and complicated for them.
